README.md

```diff
@@ -163,16 +166,28 @@ Check out the [sample app](./test-app) to see a webpack config in action or read
 - [assets](./packages/assets)
 - [babel6](./packages/babel6)
 - [dev-server](./packages/dev-server)
+- [elm](./packages/elm)
 - [extract-text](./packages/extract-text)
 - [postcss](./packages/postcss)
 - [sass](./packages/sass)
-- [typescript](./packages/typescript)
 - [tslint](./packages/tslint)
+- [typescript](./packages/typescript)
 - [uglify](./packages/uglify)
 - [webpack](./packages/webpack)
 
+
+## Third-party blocks
+
+- [webpack-blocks-happypack](https://github.com/diegohaz/webpack-blocks-happypack) — HappyPack
+- [webpack-blocks-purescript](https://github.com/ecliptic/webpack-blocks-purescript) — PureScript
+- [webpack-blocks-server-source-map](https://github.com/diegohaz/webpack-blocks-server-source-map) — source map for server bundle
```

Comment:
Sure, I just mean that it looks a tiny bit odd. But I don't really mind, just wanted to get a 2nd opinion 😉